RecordingApp/app/src/main/java/wycliffeassociates/recordingapp/ProjectManager/activities/ActivityProjectManager.java

```diff
@@ -375,33 +390,33 @@ public void zipProgress(int progress){
         mPd.show();
     }
 
-    public void dismissProgress(){
+    public void dismissProgress() {
         mPd.dismiss();
     }
 
-    public void incrementProgress(int progress){
+    public void incrementProgress(int progress) {
         mPd.incrementProgressBy(progress);
     }
 
-    public void setUploadProgress(int progress){
+    public void setUploadProgress(int progress) {
```

Comment:
Why is this public API called `setUploadProgress` instead of just `setProgress`?